### DISCUSSION TITLE
Cannot merge PRs in repos that formerly had merge queue enabled

### DISCUSSION THREAD

**Discussion by Wevah** [community/community#63676] [Enterprise]
A (private) repo used to have the merge queue enabled before it became an enterprise feature. When trying to merge a PR on the web, the merge button correctly says "Merge pull request" and works correctly. In the app, the button still says "Add to merge queue" (or whatever), and when pressed and confirmed an alert appears that says the repo doesn't have the merge queue enabled.

(At least, I'm assuming the former queue-enabled status is what's causing this.)
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.126.0 (107057637)
Device: iPhone 13 Pro Max (iOS 16.6)
</details>

**Comment by eliperkins** [comment]
Hey @Wevah, thanks for this bug report!

I've created an issue in our internal repo to track this bug. I'll follow up with our engineering team to get the root cause of this and solve it for as soon as possible. 🙇

**Comment by aconsuegra** [comment]
Hi @Wevah, thanks for the report. We are looking at this issue now, but need a bit more information to debug. Do you see this issue with all pull requests in that repository? Even with the ones that are created after disabling the merge queue feature? 

In addition, could you try logging out from the application and logging in again?

**Reply by Wevah** [reply]
I *think* PRs made later exhibited this issue, but I also think I was able to merge a recent PR, so this may be resolved for me. (Apologies for the late reply.)
